### PR TITLE
Arabic translation

### PR DESCRIPTION
**Pull request by fady-a-m-ibrahim**
#### Is your pull request related to a problem? Please describe
The PR related to the issue Internalization: Add your local language #57

#### Describe the solution you'd like
Add the Arabic translation to src/locales/ar/LC_MESSAGES/messages.po and src/services/profile.py.

#### Test plan
There are no new tests added.

### CODE REVIEW

**Comment by fady-a-m-ibrahim** [comment]
Thank you @aboullaite for your comment.
Could you please give me an example of a word to word translation and the semantic translation that should be.

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+"😎لا تقلق!\n"
+" يسمح لك التحكم في الإصدار بإدارة التغييرات على الملفات بمرور الوقت  ️.⏱ "
+
+#: services/messenger.py:271
+msgid ""
+"You can use version control to version code, binary files, and digital "
+"assets 🗄️."
+msgstr ""
+" يمكنك استخدام ادوات  التحكم في الإصدار لتعيين إصدار  ملفات شفرة المصدر "
+"وملفات الصيغة الثنائية (لغة الألة) بالإضافة إلى الأصول الرقمية  .🗄️ "
```
Comment (aboullaite) [inline src/locales/ar/LC_MESSAGES/messages.po:166]: - Can you add more ponctuations ? 
- I prefer the use of `التحكم بالمراجعات.` for version control
- You can use `كود` for code! شفرة المصدر is a bit vague

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+#: services/messenger.py:370 services/messenger.py:711
+msgid "Good question 👌🏽"
+msgstr " سؤال جيد  👌🏽 "
+
+#: services/messenger.py:374 services/messenger.py:715
+msgid ""
+"Git is a type of version control system (VCS) that makes it easier to "
+"track changes to files. "
+msgstr ""
+" Git  هو نوع من أنواع نظم التحكم في الإصدارات  (VCS)  يُسهل عليك تعقب "
```
Comment (aboullaite) [inline src/locales/ar/LC_MESSAGES/messages.po:237]: Just remove the word VCS! doesn't really make sense here

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,623 @@
...
+"هذا المستودع على جهاز الكمبيوتر الخاص بك. "
+
+#: services/messenger.py:487
+msgid ""
+"Let's create a clone of your fork locally on your computer.\n"
+"On GitHub, in your newly forked project, Click on `Code` above the list "
+"of files"
+msgstr ""
+" لنقم بإنشاء نسخة من تفريعتك على جهاز الكمبيوتر الخاص بك.  \n"
+"  على  GitHup   ، في مشروعك الجديد المتفرع ، انقر فوق \"الرمز\" أعلى جدول"
```
Comment (aboullaite) [inline src/locales/ar/LC_MESSAGES/messages.po:327]: you should keep `Code`, as the image contains it...

File: src/locales/ar/LC_MESSAGES/messages.po (hunk 2 above)
Comment (fady-a-m-ibrahim) [reply src/locales/ar/LC_MESSAGES/messages.po]: Done

File: src/locales/ar/LC_MESSAGES/messages.po (hunk 3 above)
Comment (fady-a-m-ibrahim) [reply src/locales/ar/LC_MESSAGES/messages.po]: Done

File: src/locales/ar/LC_MESSAGES/messages.po (hunk 1 above)
Comment (fady-a-m-ibrahim) [reply src/locales/ar/LC_MESSAGES/messages.po]: I have added more punctuations. 
I have used ملفات الأكواد
I feel التحكم في الإصدار is nearer to the meaning, maybe I'm wrong.

Comment (fady-a-m-ibrahim) [reply src/locales/ar/LC_MESSAGES/messages.po]: Thank you @aboullaite I have replied to all your comments and update the translation.
Also, I think
version is إصدار
and
revision is مراجعة

**Comment by fady-a-m-ibrahim** [comment]
Hello @aboullaite, Hoping you are doing fine.
There is an update, I have resolved the conflict in src/services/profile.py
Great Thank you,

**Comment by aboullaite** [comment]
Sounds good! Also, all files should have this license header
```
# Copyright (c) Facebook, Inc. and its affiliates.
#
# This source code is licensed under the MIT license found in the
# LICENSE file in the root directory of this source tree.
```

**Comment by fady-a-m-ibrahim** [comment]
Thank you @aboullaite for your support. 
I have added the license header.

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -0,0 +1,628 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Arabic translations for PROJECT.
```
Comment (elinguiuriel) [inline src/locales/ar/LC_MESSAGES/messages.po:6]: @fady-a-m-ibrahim a file can only have one copyright, please remove this.

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+# Copyright (C) 2020 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR Fady Ibrahim, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
```
Comment (elinguiuriel) [inline src/locales/ar/LC_MESSAGES/messages.po:13]: You can put the project name here `Arabic translation`

File: src/locales/ar/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2020-08-15 20:37+0200\n"
+"PO-Revision-Date: 2020-08-15 12:58+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (elinguiuriel) [inline src/locales/ar/LC_MESSAGES/messages.po:17]: You can put your name and address here !

**Comment by fady-a-m-ibrahim** [comment]
Thank you @elinguiuriel 
I have removed the second copyright.

**Comment by elinguiuriel** [comment]
> Thank you @elinguiuriel
> I have removed the second copyright.

@fady-a-m-ibrahim LGTM, @aboullaite will have a last look as an Arabic speaker and merge your PR, very nice work 🥇

**Comment by aboullaite** [comment]
Thank you for bearing with us  @fady-a-m-ibrahim through this PR! Great work

**Comment by fady-a-m-ibrahim** [comment]
Thank you @aboullaite for all your support and nice comments. I do appreciate it.

**Comment by elinguiuriel** [comment]
@fady-a-m-ibrahim make sure to set Arabic as your facebook language and test the chatbot here : https://m.me/chatbot.com

After many challenges #122 and #123, I was able to make Arabic works

image

image

**Pull request merged**